CHANGELOG.md

```diff
@@ -5,10 +5,14 @@
 ### Changed
 
 - Always enable verbose (`-v`) flag when a log directory is configured (`coder.proxyLogDir`).
+- Replaced SSE paths with a one-way WebSocket and centralized creation on `OneWayWebSocket`, and unified socket handling.
 
 ### Added
 
 - Add support for CLI global flag configurations through the `coder.globalFlags` setting.
+- Add logging for all REST and some WebSocket traffic, with REST verbosity configurable via `coder.httpClientLogLevel` (`none`, `basic`, `headers`, `body`).
```

Comment:
We mention the web socket logs in a point below so I think mentioning here is redundant. 

```suggestion
- Add logging for all REST traffic. Verbosity is configurable via `coder.httpClientLogLevel` (`none`, `basic`, `headers`, `body`).
```